src/cryptography/hazmat/backends/openssl/x509.py

```diff
@@ -226,6 +226,10 @@ def __eq__(self, other):
     def __ne__(self, other):
         return not self == other
 
+    def __hash__(self):
+        # TODO: Using fingerprint() with SHA256 is way overkill.
```

Comment:
Is there a way to access the Python siphash impl? What's the long term strategy?